src/lib/services/verification/evm/api.ts

```
@@ -11,13 +18,97 @@ export const getEvmVerifyConfig = async () =>
+  evmVersion,
+  contractLibraries,
+}: SubmitEvmVerifyFlattenArgs) => {
+  const settings = {
+    evmVersion: evmVersion === "default" ? "cancun" : evmVersion,
+    optimizer: {
+      enabled: optimizerConfig.enabled,
+      runs: Number(optimizerConfig.runs),
+    },
+    libraries: contractLibraries.value.reduce(
+      (acc, library) => ({
+        ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)